addons/account/models/company.py

```diff
@@ -259,6 +259,13 @@ def _compute_invoice_terms_html(self):
             if html:
                 company.invoice_terms_html = html
 
+    def _get_fiscalyear_last_day(self, year):
+        """ Return a last day of the month based on year."""
+        if not calendar.isleap(year) and self.fiscalyear_last_month == "2" and self.fiscalyear_last_day == 29:
```

Comment:
I don't think you need the check `isleap` (as if we need the 29/02, then it's always the last day of the month)